examples/azure/terraform-azure-mysqldb-example/main.tf

```diff
@@ -33,6 +33,10 @@ resource "random_password" "password" {
   length           = 16
   special          = true
   override_special = "_%@"
+  min_upper        = "1"
```

Comment:
please have these password changes in a separate PR since they are not related , same for terraform-azure -sql-example/main.tf below

Reply:
Removed